core_contracts/governance/governance.py

```diff
@@ -90,34 +173,53 @@ def cancelVote(self, name: str) -> None:
         proposal.status.set(ProposalStatus.STATUS[ProposalStatus.CANCELLED])
 
     @external
-    @only_owner
-    def defineVote(self, name: str, description: str, quorum: int, vote_start: int, duration: int, snapshot: int, actions: str,
-                   majority: int = MAJORITY) -> None:
+    def defineVote(self, name: str, description: str, vote_start: int, duration: int, 
+                   snapshot: int, actions: str) -> None:
         """
-        Names a new vote, defines quorum, and actions.
+        Define a new vote and which actions are to be executed if it is successful.
+
+        Parameters:
+        name         -   Name of the vote.
+        description  -   Description of the vote.
+        vote_start   -   Day to start the vote.
+        duration     -   Number of days the vote will be active.
+        snapshot     -   Which day to use for the baln stake snapshot.
+        actions      -   Json string on the form:  {'<action_1>': {<kwargs for action_1>},
+                         {'<action_2>': {kwargs_for_action_2}}, ...}
         """
         if len(description) > 500:
             revert(f'Description must be less than or equal to 500 characters.')
-        if not 0 < quorum < 100:
-            revert(f'Quorum must be greater than 0 and less than 100.')
-        if vote_start <= self.getDay():
+        if vote_start < self.getDay():
```

Comment:
vote_start is the day on which voting is first allowed so it also can't be equal to the current day, which started at the last snapshot time, so the comparison here needs to be <=.

Reply:
Not sure I understand the reason here. You do not want users to be able to define a vote which starts at the same day as it was defined on? thus the <=? Is that understanding correct?

Reply:
Yeah, so the day change is at 17:00 UTC, if someone defines a vote at 23:00 UTC and we accept a vote_start equal to the current day, then 6 hours of voting time would already have passed. It is a little confusing and I am not sure how we could make it clearer. I think the difficulty comes from the snapshot time being at the end of a day, but if you say the vote starts on a given day it makes sense that it starts at the beginning of the day, which is the time of the snapshot for the previous day. I guess we could say the vote starts at a particular snapshot time, so if a vote is defined to start at snapshot 102 voting would actually be allowed when day 103 starts. As it is when you look at the data for a vote definition it shows an end day and votes are not accepted on that day, sort of like the range function in Python.

Reply:
Ah, I see. Thank you for the detailed explanation.